src/pages/search/index.js

```diff
@@ -77,76 +83,134 @@ function SearchPage () {
 
     return (
         <>
-
-        <div className="parent">
-            <BackButton />
-            <h1>Search Results</h1>
-            <section className="searchparent">
-                <div className="">
-                    {formData.length !== 0 
-                    ? (<form onSubmit={onSubmit}>
-                    <TextInput
-                    value={formData}
-                    onChange={onChange}
-                    name={'searchusers'}
-                    type="text"
-                    icon={<SearchIcon />}/>
-
-                    <button className="clearbutton">
-                        <CrossBlackIcon/>
-                    </button>
-
-                    
-                    </form>) : (
-                        <form onSubmit={onSubmit}>
-                        <TextInput
-                        value={formData}
-                        onChange={onChange}
-                        name={'searchusers'}
-                        type="text"
-                        icon={<SearchIcon />}/>
-                        
-                        </form>
-                    )}
-
+            {userRole === 'TEACHER' ? (
```

Comment:
There seems to be a lot of duplication here.

Could you find a way to do something like:
```
 { 
    userRole === 'Teacher' && (
      <div>Add Note</div>
      <div>Move to Cohort</div>
    )
}
```